pkg/pod/script_test.go

```diff
@@ -225,7 +225,9 @@ script-3`,
 		VolumeMounts: preExistingVolumeMounts,
 		Args:         []string{"my", "args"},
 	}}, []v1beta1.Sidecar{}, &v1beta1.TaskRunDebug{
-		Breakpoint: []string{breakpointOnFailure},
+		Breakpoints: &v1beta1.TaskBreakpoints{
```

Comment:
Can we test for all kind of breakpoints?

Reply:
yeah~ I modified the function `TestConvertScripts_WithBreakpoints` to test all breakpoints.